src/components/ToDoList/ToDoList.jsx

```diff
@@ -6,31 +6,81 @@ import { Button, InputGroup, FormControl } from 'react-bootstrap';
 class ToDoList extends React.PureComponent {
 
   state = {
+    inputEditMessage: '',
     inputTaskValue: '',
     tasks: []
   }
 
   componentDidMount() {
     axios.get(`http://127.0.0.1:3001/api/tasks`).then(res => {
-      this.setState({ tasks: res.data });
+      this.setState({
```

Comment:
You don't need to set `false` explicitly. `editMode` is `undefined` if it is not set. `undefined` is the same as `false`.